src/kibana/plugins/discover/components/field_chooser/field_chooser.html

```diff
@@ -104,16 +104,25 @@
     </div>
   </div>
 
-  <ul bindonce
+  <ul
+    bindonce
```

Comment:
This looks like a simple re-organization of this section, but is `bindonce` actually used anywhere here? Should it be removed?